### DISCUSSION TITLE
Is there any way add custom annotations to all milvus components (+dependencies also) ?

### DISCUSSION THREAD

**Discussion by SeunghyunYoo** [milvus-io/milvus#28196] [Q&A and General discussion]
There is already other metric exporter In my k8s cluster, so i just want to extract metric from milvus component using `metric API`

Is there any way add custom annotations to all milvus components (+dependencies also), when i construct milvus cluster using milvus-operator ?

the milvus operator version: 8.4.0, milvus version: 2.3.1

**Comment by LoveEachDay** [comment]
You can add podAnnotations for each component like this:
```
spec:
  spec:
    dataNode:
      podAnnations:
          a: b
```

**Comment by SeunghyunYoo** [comment]
thxs :)
